packages/ui-components/src/helpers/tailwind.ts

```diff
@@ -19,7 +19,7 @@ export function markClassesUsed(classes: string[]) {
  */
 export enum TailwindBreakpoints {
   sm = 640,
-  md = 746,
+  md = 768,
```

Comment:
I had to change it because it didn't match the 'md' value in Tailwind, meaning if you have to use both JS and CSS classes for responsiveness there was a gap between `746` and `768` that would be the `md` breakpoint in JS but not yet in CSS